### ISSUE TITLE
Add code for Facial Points recognition using CNN

### ISSUE THREAD

**Issue by manavkapadnis** [DSCVITBHOPAL/ML-Reserve#57]
**Project Description**: Provided an image of the face of a person, the trained 5 layered CNN model accurately predicts positions of certain features of the face such as left eye center, right eye center, left eyebrow center, right eyebrow center, nose tip, mouth left corner, mouth right corner, mouth center top lip, mouth center bottom lip and others.

**Model training Parameters**:
<br><br>
Epochs = 50<br>
LR = 1e-5 (default value)<br>
Batch_size = 256<br>
Optimizer = Adam<br>
Loss Function = Mean Squared Error Loss<br>
Metric = Mean Absolute Error<br>
<br>

**Comment by manavkapadnis** [comment]
@JackJJCodes  please have a look at the PR, I have updated the readme as well as the PR description with the project details..
